.github/workflows/mdbook.yml

```diff
@@ -0,0 +1,102 @@
+name: mdBook
+
+on:
+  push:
+    branches:
+      - main
+      - rzadp/rfc-book
+  # schedule:
+  #   - cron: "0 0 * * *" # Once a day
+
+jobs:
+  mdbook:
+    runs-on: ubuntu-latest
+    permissions:
+      # For writing to gh-pages branch.
+      contents: write
+    steps:
+      - name: Checkout this repo
+        uses: actions/checkout@v3
+
+      - name: Precreate necessary directories
+        run: |
+          mkdir -p mdbook/src/proposed
+          mkdir -p patches/text
+          
+      - name: Download all proposed RFCs (open PRs)
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const owner = 'polkadot-fellows'
+            const repo = 'RFCs'
+            const prs = await github.paginate(github.rest.pulls.list, {owner, repo, state: 'open'})
+
+            /*
+              The open PRs are potential proposed RFCs.
+              We iterate over them and filter those that include a new RFC markdown file.
+            */
+            for (const pr of prs) {
+              const addedMarkdownFiles = (
+                await github.rest.pulls.listFiles({
+                  owner, repo,
+                  pull_number: pr.number,
+                })
+              ).data.filter(
+                (file) => file.status === "added" && file.filename.startsWith("text/") && file.filename.includes(".md"),
+              );
+              if (addedMarkdownFiles.length !== 1) continue;
```

Comment:
This is to filter the markdowns? There should only be one file per PR?

Reply:
Exactly, there is a [process](https://github.com/paritytech-stg/RFCs#process) for creating the RFC PRs and here we try to filter out any PR that is not a standard RFC PR.